Resources/Prototypes/Nyanotrasen/Entities/Decoration/snowman.yml

```diff
@@ -0,0 +1,34 @@
+- type: entity
+  id: Snowman
+  parent: BaseStructure
+  name: snowman
+  description: An adorable friend. 
+  abstract: true
+  components:
+  - type: Transform
+    noRot: true
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Inflatable
+  - type: PlaceableSurface
```

Comment:
PlaceableSurface lets you put things on top of another entity. Was this intentional? I'd have to check if it works with BaseStructure's default collision mask.